test/jdk/javax/swing/JEditorPane/8328953/EditorPaneCharset.java

```diff
@@ -39,11 +32,20 @@
 import javax.swing.SwingUtilities;
 import javax.swing.text.Document;
 
+/*
+ * @test
+ * @key headful
+ * @bug 8328953
+ * @summary JEditorPane.read throws ChangedCharSetException
+ * @run main EditorPaneCharset
+ */
+
 public final class EditorPaneCharset {
     private static final String HTML_CYRILLIC =
             "<html lang=\"ru\">\n" +
             "<head>\n" +
-            "    <meta http-equiv=\"Content-Type\" content=\"text/html; charset=windows-1251\">\n" +
+            "    <meta http-equiv=\"Content-Type\" content=\"text/html; charset=
+			windows-1251\">\n" +
```

Comment:
Is it a valid Java syntax? The file does not compile.

I would prefer wrapping at attribute boundary rather than a value of an attribute.

No tabs are allowed in OpenJDK source code, replace with four spaces.